framework/include/utils/MooseADWrapper.h

```diff
@@ -9,6 +9,9 @@
 #include "libmesh/vector_value.h"
 #include "libmesh/tensor_value.h"
 
+#include "metaphysicl/numberarray.h"
```

Comment:
Are you sure these need to be in this header?

I would love to keep down the number of times we include these because they are big....